src/htcondor_es/convert_to_json.py

```diff
@@ -1,16 +1,17 @@
 #!/usr/bin/python
```

Comment:
this should always be in a form of
```
#!/usr/bin/env python
```
otherwise system python will be used by default.